packages/eslint-plugin-react-native/src/docs/rules/i18n-avoid-global-imports.md

```diff
@@ -0,0 +1,37 @@
+# Enforces i18n usage by avoiding global imports (@getluko/react-native/i18n-avoid-global-imports)
+
+Allows you to enforce i18n by avoiding global imports. It can cause some errors on translation rendering.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```jsx
+  // bad ❌
+  const key = i18n.t('key');
+  return (
+    <Text>{key}</Text>
+  )
+```
+
+Examples of **correct** code for this rule:
+
+```jsx
+    // good ✅
+    const getKey = () => i18n.t('key');
+    return (
+      <Text>{getKey()}</Text>
+    );
```

Comment:
I'm still not convinced about the duplicate documentation. Not a big issue with 2 rules but with a lot of rules it will be harder to maintain

Reply:
Okey let's remove these duplicated markdowns then